modules/ROOT/pages/api-groups-sla-tiers-faqs.adoc

```diff
@@ -0,0 +1,30 @@
+= SLA Tiers For API Groups FAQs
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+*Question: What happens if an API Group is published without any SLA tiers?*
+
+*Answer:*
```

Comment:
GLOBALLY REFORMAT per my Edit Tip last week on FAQS. See Style Reference.
I am sad that you missed that Slack message.

Reply:
Nope, this was meant to be removed so did not spend time on it.